src/main/java/org/opensearch/flowframework/common/FlowFrameworkSettings.java

```diff
@@ -129,6 +133,25 @@ public class FlowFrameworkSettings {
         Setting.Property.NodeScope
     );
 
+    /** This setting sets max workflows that can be simultaneously provisioned, or reprovisioned by the same tenant */
+    public static final Setting<Integer> MAX_ACTIVE_PROVISIONS_PER_TENANT = Setting.intSetting(
+        "plugins.flow_framework.max_active_provisions_per_tenant",
+        2,
+        1,
+        4,
```

Comment:
Should we also make our threadpool size dynamic and the max higher here incase we have larger nodes that can accommodate that?

Reply:
Yes.